src/services/token.service.js

```
@@ -0,0 +1,28 @@
+import { Token } from '../data/token.js';
+
+const save = async (userId, newToken) => {
```

The `save` function correctly upserts a refresh token for a user. Consider adding a small validation at the top to ensure `userId` and `newToken` are present (fail fast if not) so higher-level callers get clearer errors when called with invalid data.